File: ass45/program3.c
/*
3.Write a recursive program which display below pattern.  
Input :   5 
Output :   5 4 3 2 1 
*/
#include<stdio.h>
void Display(int x)
{
    static int i=1;
    if(x!=0)
    {
        printf("%d",x);
        x--;
        Display(x);
    }
}
int main()
{
    int iNo = 0;
    printf("Enter the Element\n");
    scanf("%d",&iNo);

    Display(iNo);
   
    return 0;
}

File: ass59/Program3.java
/*
3. Write Java program which accept String from user and display below pattern. 
Input :  Hello 
Output : H   
         H e   
         H e l   
         H e l l   
         H e l l o
*/
import java.lang.*;
import java.util.*;
class DemoString
{
	int WordFrequency(String str)
	{
		char arr[] = str.toCharArray();
		int size = arr.length;
        int i = 0, j= 0;
        
        for(i=0; i<=size-1; i++)
        {
            for(j=0; j<=i; j++)
            {
                System.out.print(""+arr[j]);
            }
            System.out.print("\n");
        }
        return 0;
    }
    
}
class Program3
{
	public static void main(String arg[])
	{
		Scanner sobj = new Scanner(System.in);
		System.out.println("Enter string");

        String str = sobj.nextLine();
        sobj.close();
		DemoString dobj = new DemoString();

		dobj.WordFrequency(str);
	}
}